### ISSUE TITLE
Is it possible to disable standard markup processing?

### ISSUE THREAD

**Issue by BwehaaFox** [nhn/tui.editor#1135]
Is there currently an option to turn off the standard processing of the contented markup type?

For example, my task does not imply the use of tables, but even if I remove the button from the navigation, no one will prevent the user from entering the table manually.

**Comment by seonim-ryu** [comment]
@BwehaaFox There is currently no option to handle the feature you mentioned. The editor follows the [gfm](https://github.github.com/gfm/#tables-extension-) specification for table, and I think you need to add an option to turn off the `gfm` specification to handle this feature. You can disable table rendering by using the `customHTMLRenderer` option, but table syntax highlighting will remain when entering a table in the Markdown Editor. So it doesn't make sense to use the `customHTMLRenderer` option and I think I need to add a new option.

**Comment by bdevg** [comment]
Are you using react?

**Comment by BwehaaFox** [comment]
No, I am using Ember.js and an [addon](https://github.com/evocount/ember-tui-editor) for it
